tests/test_talonsrx.py

```diff
@@ -571,22 +571,24 @@ def test_basemotorcontroller_setNeutralMode(talon, cdata):
 
 
 def test_basemotorcontroller_selectedSensorPosition(talon, cdata):
-    # no selected device, should not fail
+    # default selected sensor is now a quad encoder
```

Comment:
FYI, the firmware default selected sensor has actually always been a quad encoder.